keras_cv/losses/focal.py

```diff
@@ -22,7 +22,11 @@ class FocalLoss(keras.losses.Loss):
     """Implements Focal loss
 
     Focal loss is a modified cross-entropy designed to perform better with
-    class imbalance. For this reason, it's commonly used with object detectors.
+    class imbalance. It minimizes the weight of easy -ves on the loss function
```

Comment:
Since we already reference the FocalLoss paper below, I think this detail is a bit extraneous (it's not critical information to a user of this API, and if they want to learn more they're already referred to the paper)